Source/DafnyCore/AST/Statements/Verification/AssumeStmt.cs

```diff
@@ -32,4 +32,12 @@ public AssumeStmt(RangeToken rangeToken, Expression expr, Attributes attrs)
   public bool SetIndent(int indentBefore, TokenNewIndentCollector formatter) {
     return formatter.SetIndentAssertLikeStatement(this, indentBefore);
   }
+
+  public override void GenResolve(INewOrOldResolver resolver, ResolutionContext context) {
+    base.GenResolve(resolver, context);
+
+    if (!resolver.Options.Get(CommonOptionBag.AllowAxioms) && !this.IsExplicitAxiom()) {
+      resolver.Reporter.Warning(MessageSource.Resolver, ResolutionErrors.ErrorId.r_assume_statement_without_axiom, Tok, "assume statement has no {:axiom} annotation");
```

Comment:
```suggestion
      resolver.Reporter.Warning(MessageSource.Verifier, ResolutionErrors.ErrorId.r_assume_statement_without_axiom, Tok, "assume statement has no {:axiom} annotation");
```

Reply:
> ResolutionErrors to VerificationErrrors for clarity

I've moved it to `AssumeStmt`, where the warning is produced.